### ISSUE TITLE
zcu100-zynqmp is missing UART1 routing

### ISSUE THREAD

**Issue by agraf** [Xilinx/hdf-examples#1]
The zcu100-zynqmp example file does not route UART1 (the 3.3V separate UART) and instead only exposes UART0 which relies on EMIO routing.

Given the board has a physically accessible UART1, it should really also get exposed.

**Comment by mharth** [comment]
Agreed, the only reason to have it left out was because it was susceptible to board breakage (due to shorting) since it is a 3 pin connector.

**Comment by agraf** [comment]
Well, I find it very useful to be able to debug the system without any bitstream loaded ;). Currently that doesn't work.

**Comment by mharth** [comment]
Let me check with the HW team and try to get a design that will work on uart1, instead of EMIO routing

**Comment by agraf** [comment]
With the bitstream properly injected I have a working UART1 on the LS connector now, so it's not terribly urgent.
